Server/Modules/Items/Models/ItemType.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Fracture.Server.Modules.Items.Models
+{
+    public enum ItemType
+    {
+        Helmet,
+        Chestplate,
+        Gauntlets,
+        Leggings,
+        Boots,
+        Ring
+    }
```

Comment:
These are placeholders for now since I don't remember (and hadn't found any notes about it) if we had decided on what the types should be.